docker/run.sh

```diff
@@ -159,8 +159,15 @@ function _run_tests {
 
     function _test_python {
         ./manage.py create_kafka_topics
-        logmsg INFO "./manage.py test ${py_test_args[*]}"
-        ./manage.py test "${py_test_args[@]}"
+        if [ "$TRAVIS_EVENT_TYPE" ]; then
```

Comment:
This doesn't look like valid bash syntax to me. Did you intend to do
```suggestion
        if [ -n "$TRAVIS_EVENT_TYPE" ]; then
```

Reply:
I've got no problems sticking in an `-n` for clarity